### DISCUSSION TITLE
How to get the  Left ventricular ejection fraction on cardiac ultrasound from mimic iii database？

### DISCUSSION THREAD

**Discussion by liedezhanghu** [MIT-LCP/mimic-code#1267] [MIMIC-III]
How to get the  Left ventricular ejection fraction on cardiac ultrasound from mimic iii database？

**Comment by alistairewj** [comment]
I assume this is also related to your question:

> [liedezhanghu](https://github.com/liedezhanghu)
[2 hours ago](https://github.com/MIT-LCP/mimic-code/discussions/1220#discussioncomment-2316626)
I read your article about extracting echocardiography data from mimiciii written six years ago(https://github.com/MIT-LCP/echo-data). After reading the readme part in detail, I still have doubts about how to extract it. First of all, https://physionet.org/users/ this link can't be accessed, secondly,where can I get the"Echocardiography data" CSV file?Please forgive my grammar mistakes because English is not my mother language.

Unfortunately that repository was never completed! We had some work to extract structured interpretation of the echos. Perhaps we will readdress it with the upcoming release of more echo data.

Echos have two sections: the numeric values, and the free-text interpretation. Unfortunately we were only able to acquire and deidentify the free-text interpretation, so numeric values like LVEF aren't available. However, statements regarding its status are, and some NLP will get you a decent idea as to whether there is any dysfunction.

**Reply by liedezhanghu** [reply]
Thank you for your reply.  I have seen the numerical range of LVEF of patients, such as 30-40% and 40-50% in some literature. Does this description belong to the structured interpretation of echos? Is such data available now?
 We look forward to your future work, which is of great significance to us!

**Reply by DrZoggg** [reply]
Yes, I agree with you very much about the LVEF information shown in some MIMICIII-based articles. (PMID: 35146943; PMID: 34993210; PMID: 34301649, et al.) However, the ITEMID of 'ejection fraction'  includes '227008', '2699', '2697' according to D_ITEMS TABLE. However, after querying other TABLE (CHARTEVENTS; LABEVENTS, et al) with their IDs, only few patients can be retrieved with information about ejection fraction.  Even if not obtained LVEF directly, E.g. calculated by formula （EF=(EDV-ES)×100%/EDV), the indicator 'ES'  also cannot be retrieved. So  how the LVEF information to be obtained from the MIMICiii database?
